apps/condo/pages/settings/integration/[id]/index.tsx

```diff
@@ -0,0 +1,218 @@
+ `
+
+const BillingIntegrationDetailsPage = () => {
+    const intl = useIntl()
+    const ErrorMessage = intl.formatMessage({ id: 'errors.PdfGenerationError' })
+    const LoadingMessage = intl.formatMessage({ id: 'Loading' })
+    const NoPermissionMessage = intl.formatMessage({ id: 'NoPermissionToSettings' })
+    const DefaultStartButtonMessage = intl.formatMessage({ id: 'StartIntegration' })
+    const DefaultInstructionButtonText = intl.formatMessage({ id: 'Instruction' })
+    const ContinueMessage = intl.formatMessage({ id: 'property.Continue' })
+    const ConfirmModalTitle = intl.formatMessage({ id: 'BillingIntegrationContextConfirmQuestion' })
+    const CancelMessage = intl.formatMessage({ id: 'Cancel' })
+    const CreateContextRestrictionMessage = intl.formatMessage({ id:'BillingIntegrationContextNoReturns' }, {
+        buttonValue: ContinueMessage,
+    })
+    const ContextAlreadyCreatedMessage = intl.formatMessage({ id: 'ContextAlreadyCreated' })
+    const CompanyNameLabel = intl.formatMessage({ id: 'CompanyName' })
+    const AnotherContextAlreadyCreatedMessage = intl.formatMessage({ id: 'AnotherContextAlreadyCreated' }, {
+        companyName: CompanyNameLabel,
+    })
+
+    const { query, push } = useRouter()
+    const integrationId = get(query, 'id')
+
+    const userOrganization = useOrganization()
+    const organizationId = get(userOrganization, ['organization', 'id'])
+    const canManageIntegrations = get(userOrganization, ['link', 'role', 'canManageIntegrations'], false)
+
+    const {
+        obj: integration,
+        loading: integrationLoading,
+        error: integrationError,
+    } = BillingIntegration.useObject({
+        where: {
+            id: String(integrationId),
+        },
+    })
+
+    const {
+        obj: currentContext,
+        error: contextError,
+        loading: contextLoading,
+    } = BillingIntegrationOrganizationContext.useObject({
+        where: {
+            organization: {
+                id: organizationId,
+            },
+        },
+    }, {
+        fetchPolicy: 'network-only',
+    })
+
+    const sender = getClientSideSenderInfo()
+    const createContextAction = BillingIntegrationOrganizationContext.useCreate({
+        integration: String(integrationId),
+        organization: organizationId,
+        // @ts-ignore
+        state: { dv: 1, sender },
+        // @ts-ignore
+        settings: { dv: 1, sender },
+    }, () => {
+        push(SETTINGS_PAGE_ROUTE)
+    })
+
+    const { confirm } = Modal
+    const showConfirmModal = () => {
+        confirm({
+            title: ConfirmModalTitle,
+            icon: <ExclamationCircleOutlined />,
+            content: (
+                <Alert message={CreateContextRestrictionMessage} type={'warning'}/>
+            ),
+            okText: ContinueMessage,
+            cancelText: CancelMessage,
+            onOk () {
+                // @ts-ignore
```

Comment:
Why ts ignore?

Reply:
Since our project built like that. We require same interface on action declaration and on action execution. I can double all data from above, but what's the reason,  if it's not even will be overwritten ?

Reply:
add typedefs at apps/condo/domains/billing/utils/clientSchema/BillingIntegrationOrganizationContext.ts

Reply:
onOk?: (...args: any[]) => any

i'm not sure this callback should return sync function call resut

Reply:
sounds like resolved?)